albert_garcia/flask_fun/game2.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template, request, redirect, session
+import random
+app = Flask(__name__)  
+app.secret_key ='thisIsSecret'
+
+print(__name__)
+@app.route('/')
+def Index():
+    rand = random.randrange(1,101)
```

Comment:
rand should be made once and stored in session(until the guess is correct). For now rand is not available in Guess(), but if session['rand'] is assigned then it can be used for comparisons on line 16,18,20